src/rust/engine/logging/src/logger.rs

```diff
@@ -169,6 +173,19 @@ impl Log for PantsLogger {
       return;
     }
 
+    let log_msg = format!("{}", record.args());
+    {
+      let literal_filters = self.literal_filters.lock();
+      if literal_filters.iter().any(|filt| log_msg.starts_with(filt)) {
+        return;
+      }
+
+      let regex_filters = self.regex_filters.lock();
+      if regex_filters.iter().any(|re| re.is_match(&log_msg)) {
```

Comment:
Note that this was originally checking against the `log_string`, meaning it had the timestamp and `[LEVEL]` included. That was fine because of regex, but does not work for literal prefix matching.